### DISCUSSION TITLE
Ability to see all slides at once?

### DISCUSSION THREAD

**Discussion by lmajano** [ismail9k/vue3-carousel#351] [Q&A]
We have built a nice slideshow viewer with this amazing plugin.  However, we have a request to create a PDF out of our slideshow.  Is there a built-in way to show ALL slides in a stacked fashion, so we can build a PDF of all the slides in the carousel?

**Comment by ismail9k** [comment]
Currently, this feature is unavailable. However, you can explore working with two options for displaying the items: using the carousel or stacking them below each other. This way, you will be able to generate a PDF version of the content with the stacked display.
